install/boards/bcm_27xx.sh

```diff
@@ -38,7 +38,7 @@ for STRING in \
     "dtoverlay=uart4" \
     "dtoverlay=uart5" \
     "dtparam=i2c_vc=on" \
-    "dtoverlay=i2c1,baudrate=1000000" \
+    "dtparam=i2c_arm_baudrate=1000000" \
```

Comment:
![image](https://github.com/bluerobotics/BlueOS-docker/assets/4013804/91f25c72-5de7-430f-8c01-7f894abfd50e)

```suggestion
    "dtoverlay=i2c1" \
    "dtparam=i2c_arm_baudrate=1000000" \
```